### ISSUE TITLE
Koreader doesn't refresh

### ISSUE THREAD

**Issue by bbyucca** [ddvk/remarkable-autoinstall#13]
Using koreader the page only refresh after a couple of actions resulting in menu's en letters displayed all on top of each other.  My remarkable is on version 2.5.1.45. I did used toltec earlier, but made sure rm2fb and remux was removed.  I read the koreader is now stable version?  Any suggestions, advice?

**Comment by ddvk** [comment]
strange, are you sure there are no multiple instances running?

**Comment by bbyucca** [comment]
Currently I can't recreate the situation. I got the toltec route working, but I'm not happy about it. So I might try this autoinstall again.  When I do I'll let you know if I run into the same issue.

**Comment by bbyucca** [comment]
Okay I've tried this again and it is working as it should. I guess all the tinkering with the device and toltec made it go wonky. This time I made sure the rM2 was rebooted after I removed rm2fb and remux packages. I like the simplicity of this autoinstall, it's inconvenient that it isn't multitasking like with remux, but I experienced too much vague issues with swiping with remux running. Perhaps it will be better in the future or you might come up with a multitasking solution ;-)
